### ISSUE TITLE
Does this api still work?

### ISSUE THREAD

**Issue by parkershamblin** [cryptocontrol/python-api#1]
Hi,

I installed and set up everything as instructed, however, when I run the sample code I receive the error
```
if (response.status_code is not 200): raise Exception("Bad response from the CryptoControl API")
Exception: Bad response from the CryptoControl API
```

I see the website is still up and running, however, I'm assuming the python API is not. If I'm wrong please let me know because I'd love to use this repo.

**Comment by toppylawz** [comment]
I don't think it still working. Please if you come across any working one similar to this, kindly drop it here.

**Comment by dhyaan-se** [comment]
No it's not working
